src/streaming/protection/models/ProtectionModel_01b.js

```diff
@@ -182,6 +182,14 @@ function ProtectionModel_01b(config) {
         // Replacing the previous element
         if (videoElement) {
             removeEventListeners();
+
+            // VUDU Rik - close any open sessions
+            if (sessions) {
```

Comment:
according to me, sessions doesn't need to be tested, because it is inialized as an empty array.

Reply:
@vuduDASH, it would be also nice if you remove the company/name part from the comment line. 

We will use git blame in case an issue is raised because of this and we need to look for the guilty one. Just kidding :).